Dockerfile.ci

```diff
@@ -3,16 +3,17 @@ FROM elifesciences/proofreader-php:latest AS proofreader
 FROM elifesciences/annotations_cli:${commit}
 USER root
 
-# extract into another image
-RUN git clone https://github.com/asm89/smoke.sh /opt/smoke.sh
-
 RUN mkdir -p build && chown www-data:www-data build
 
 USER elife
 COPY --from=proofreader --chown=elife:elife /srv/proofreader-php /srv/proofreader-php
 RUN ln -s /srv/proofreader-php/bin/proofreader /srv/bin/proofreader
 
+USER root
+RUN composer --no-interaction install --no-suggest --classmap-authoritative
+
 USER www-data
```

Comment:
Files should be `elife`?

Reply:
Still experimental, but seemed to work in `hypothesis-dummy` https://github.com/elifesciences/hypothesis-dummy/blob/master/Dockerfile#L3-L8
Would change it as a separate PR because it influences all the `COPY` instructions which need to specify explicitly the owner, which would overly confuse the diff of the `Dockerfile`s here